.github/workflows/create-release-tags.yml

```diff
@@ -0,0 +1,35 @@
+  # Releases always publish a `v[major].[minor].[patch]` tag
+  # This action creates/updates `v[major]` and `v[major].[minor]` tags
+  name: Create release tags
```

Comment:
I think we need special handling for pre-releases here? this should not run for those I guess. Ideally we only run this if the version matches `^(\d+)\.(\d+)\.(\d+)$`, I suppose?

Reply:
ah, if that is the case, then all good!